sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpClient.cs

```diff
@@ -515,7 +515,7 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
 
                 if (ManagementLink?.TryGetOpenedObject(out var _) == true)
                 {
-                    await ManagementLink.CloseAsync().ConfigureAwait(false);
+                    await ManagementLink.CloseAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
Out of curiosity, why did we need to change this? Were the old methods removed?

Reply:
The new overload triggered an error from our analyzer about passing along the cancellation token.   But, since we don't want to cancel this deep in to avoid an inconsistent state - it's just `None`.